src/OrbitQt/MultipleOfSpinBox.h

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+#ifndef ORBIT_QT_Q_MULTIPLE_OF_EIGHT_SPIN_BOX_H_
+#define ORBIT_QT_Q_MULTIPLE_OF_EIGHT_SPIN_BOX_H_
+
+#include <QSpinBox>
+
+namespace orbit_qt {
+
+// A specialization of QSpinBox that only accepts values that are a multiple of the `singleStep`
+// size. E.g. a `MultipleOfSpinBox` with a single step size of 8, only accepts values that are
+// a multiple of 8.
+class MultipleOfSpinBox : public QSpinBox {
+ public:
+  MultipleOfSpinBox(QWidget* parent) : QSpinBox(parent) {}
```

Comment:
`explicit` and it's common in Qt code to set a default of `nullptr` for the parent.

Reply:
Done.